api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java

```diff
@@ -10,11 +10,13 @@
     public static FlagDescriptor DUE_FOR_FIRST_VIRAL_LOAD = new FlagDescriptor() {
         @Override
         public String criteria() {
-            return "SELECT p.patient_id, DATE_FORMAT(DATE_ADD(o.value_datetime, INTERVAL 6 MONTH), '%d.%b.%Y') FROM patient p " +
-                    " INNER JOIN obs o ON p.patient_id = o.person_id " +
-                    " INNER JOIN encounter e ON o.encounter_id = e.encounter_id " +
-                    " INNER JOIN encounter_type et ON e.encounter_type = et.encounter_type_id " +
-                    " WHERE ((o.concept_id = 99161 AND o.voided = FALSE AND e.voided = FALSE AND ((CURRENT_DATE() BETWEEN DATE_ADD(o.value_datetime, INTERVAL 5 MONTH) AND DATE_ADD(o.value_datetime, INTERVAL 6 MONTH)) AND et.uuid='8d5b27bc-c2cc-11de-8d13-0010c6dffd0f'))) AND o.person_id NOT IN (SELECT oo.person_id FROM obs oo WHERE oo.concept_id = 1305 AND oo.voided = FALSE)";
+            return "SELECT p.patient_id, DATE_FORMAT(DATE_ADD(o.value_datetime, INTERVAL 6 MONTH), '%d.%b.%Y') FROM patient p \n" +
```

Comment:
@solemabrothers 

1. What is the performance hit for joining to the person table and selecting person.dead = false?

2. No need to put false in double quotes since its a boolean 

3. When checking the transfer out why not combine it with the viral load check so that u do not have to run two separate subqueries?

FYI this feedback applies to all the queries, like I always say its better to do one query and we get it right before doing all of them